cli/tests/pcluster/cli/configure/test_pcluster_configure/test_subnet_automation_no_awsbatch_no_errors_empty_vpc/pcluster.config.yaml

```diff
@@ -1,6 +1,6 @@
 Region: eu-west-1
 Image:
-  Os: centos7
```

Comment:
should some of this be replaced with alinux2023?

Reply:
These are unit tests. I thought it would be easy to mock stuffs. In integration tests, I diversified the OSes